libraries/Adapters/Microsoft.Bot.Builder.Adapters.WeChat/Storage/IWeChatStorage.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.Bot.Builder.Adapters.WeChat
```

Comment:
Namespaces should match folder name, rename to Microsoft.Bot.Builder.Adapters.WeChat.Storage